ios/engine/KMEI/KeymanEngine/Classes/Keyboard/KeyboardScaleMap.swift

```diff
@@ -168,13 +169,13 @@ class KeyboardScaleMap {
    * the largest device smaller than or equal to the detected dimensions.
    */
   private static func getUnknownDeviceMapping(screenSize _screenSize: CGSize = UIScreen.main.bounds.size, asPhone: Bool? = nil) -> Device {
-    // Shouldn't happen, but just in case.
-    if _screenSize == CGSize.zero {
-//      // This would notify us whenever new devices are out that we haven't build a mapping for.
-//      SentryManager.captureAndLog("Cannot detect device dimensions; defaulting to smallest device for form factor.", sentryLevel: .info)
-
+      // Shouldn't happen, but just in case.
+      if _screenSize == CGSize.zero {
+      // This would notify us whenever new devices are out that we haven't build a mapping for.
       // We haven't actually updated things here in a while, so we'll just breadcrumb for now.
-      SentryManager.breadcrumbAndLog("Cannot detect device dimensions; defaulting to smallest device for form factor.", sentryLevel: .error)
+      let message = "Cannot detect device dimensions; defaulting to smallest device for form factor."
+      os_log("%{public}s", log: KeymanEngineLogger.ui, type: .error, message)
+      SentryManager.breadcrumb(message, sentryLevel: .error)
```

Comment:
Note that this won't actually notify us because we need to send a Sentry event rather than a breadcrumb -- a breadcrumb only gets sent to us attached to an event.

Reply:
I'm not sure what to do with this one. Do you think we want an event generated in this case? If there is a device that we have not build a mapping for, then we could suddenly generate more events than we would like.

Reply:
I don't think that's a problem. Sentry manages and consolidates repeated events, so that gives us a signal as to the magnitude of the issue also.